sqlalchemy_hana/dialect.py

```diff
@@ -230,16 +230,23 @@ def set_isolation_level(self, connection, level):
                     cursor.execute("SET TRANSACTION ISOLATION LEVEL %s" % level)
 
     def get_isolation_level(self, connection):
-        with connection.cursor() as cursor:
-            cursor.execute("SELECT CURRENT_TRANSACTION_ISOLATION_LEVEL FROM DUMMY")
-            result = cursor.fetchone()
+        # In this case the 'with' statement on connection.cursor() as cursor is not 
```

Comment:
Another pull request improved this function already and also leads to the merge conflict.